tox.ini

```diff
@@ -138,18 +124,3 @@ setenv =
 commands =
     python -m synmark {posargs:}
 
-[testenv:check_codestyle]
-extras = lint
-commands =
-    python -m black --check --diff {[base]lint_targets}
```

Comment:
Just curious why there is no longer a `--diff` flag in the call to black in the new setup: https://github.com/matrix-org/backend-meta/blob/2001ab8d60a922b0d1f48bd5f3b38765e9f74113/.github/workflows/python-poetry-ci.yml#L22

Reply:
No particular reason: that's an oversight. (I'm not sure it's hugely useful to have the --diff, but I'll fix that up)

Reply:
https://github.com/matrix-org/backend-meta/blob/9588c2bbc156aa62ac09d6663335b282da625a30/.github/workflows/python-poetry-ci.yml#L22 now includes --diff. The v1 tag has been moved forward to point to this. (Which is normally bad practice, but it's what's [recommended for actions](https://docs.github.com/en/enterprise-server@3.4/actions/creating-actions/about-custom-actions#using-release-management-for-actions).)